### PR TITLE
Update Regions and Versions

### PR DESCRIPTION
**Pull request by BNTK**
This update makes Regions and Versions look a bit nicer, etc...

### CODE REVIEW

**Comment by BNTK** [comment]
I may need to add EU and US and JPN reigons but other than that it looks nice

**Comment by BNTK** [comment]
hopefully that fixes some stuff

File: README.md [hunk 1]
```diff
@@ -19,8 +19,11 @@ If all three boxes are checked (only USA atm), then put [otherapp.bin](https://s
 
 ## Regions and Versions
 
-I only had a USA N3DS on 11.2, so I tested on that version. Other regions coming
-soon.
+| Version | N3DS | O3DS | 2DS |
+| --- | --- | --- | --- |
+| US 11.2 | ✓  | ✗  | ✗  |
+| EU 11.2 | ✗  | ✗  | ✗  |
+| JPN 11.2 | ✓  | ✗  | ✗  |
```
Comment (iggyvolz) [inline README.md]: Thought it was only working for US N3DS, not Japanese.

Comment (BNTK) [inline README.md]: wait. shit sec

Comment (BNTK) [inline README.md]: my bad fixed

**Comment by BNTK** [comment]
fixed JPN

**Comment by BNTK** [comment]
ready to commit i think

**Pull request merged**